src/content/guides/farmlands-portal.mdx

```diff
@@ -195,55 +197,69 @@ On the Settings page, you can change your business information. Please note this
-
-**We received a message on the Portal stating the “Barcode is invalid.” What does this mean?**
-
-The Card number being entered is either not a valid number or the Card being presented is no longer active on the Farmlands system. Check the correct Card number has been entered or request another method of payment.
-
-**How do I manage who has access to the Authorisation Portal?**
-
-User management and access is at the discretion of the Card Partner.
-
-**We have a regular customer. Do we still need to obtain an authorisation?**
-
-All sales where a Farmlands Card has been provided as payment require an authorisation prior to submitting an invoice to Farmlands.
-
-If you hold your customer’s Farmlands’ Card number on file and have their permission to charge this on a recurring basis, please select the 'Cardholder Not Present' option at the time of sale.
-
-**What if the barcode will not scan or is obscured?**
-
-The 9-digit Card number can be entered manually. This is printed on the back below the barcode and on the front of the Card under the Cardholder's name.
-
-If neither is visible, please request another method of payment. Advise the Cardholder to contact Farmlands to request a new Card.
-
-**The authorisation request has prompted us to “Check ID.” The customer is not the actual Cardholder but one of the workers of the Cardholder. Can I accept the Card?**
-
-No, unfortunately you must only accept the Card if the true owner of the Card is making the purchase.
+<AccordionItem client:visible style="border-bottom-width:1px">
+  <span slot="title">**What do we do if a customer returns one or more items, and the invoice has already been sent to Farmlands?**</span>
+  <span slot="description">Raise a credit note as normal and insert the original authorisation short code on the invoice. This will enable us to identify the original invoice and customer account details.</span>
+</AccordionItem>
+
+<AccordionItem client:visible style="border-bottom-width:1px">
+  <span slot="title">**What do we do if the customer cancels their order <u>after</u> we have sent the invoice to Farmlands?**</span>
+  <span slot="description">Raise a credit note as normal and insert the original authorisation short code on the invoice. This will enable us to identify the original invoice and customer account details.</span>
+</AccordionItem>
+
+<AccordionItem client:visible style="border-bottom-width:1px">
+  <span slot="title">**What do we do if the customer cancels their order <u>before</u> we have sent the invoice to Farmlands?**</span>
+  <span slot="description">If the entire order has been cancelled, release the hold on the customer’s funds via the Portal. There is no requirement to send an invoice to Farmlands.</span>
+</AccordionItem>
+
+<AccordionItem client:visible style="border-bottom-width:1px">
+  <span slot="title">**What do we do if we need to amend the value of the order prior to sending the invoice to Farmlands?**</span>
+  <span slot="description">
```

Comment:
@laurencemcknight Interested in what the DOM looks like here - `<div>` might be more appropriate to wrap slot content than a span

Reply:
I have removed these HTML elements, so I'm going to mark this as resolved.